operator/pkg/parsed/module.go

```diff
@@ -88,5 +88,6 @@ func (m *Module) UpdateStatusFromCluster(ctx context.Context, clnt client.Client
 
 	m.Unstructured.Object["status"] = unstructuredFromServer.Object["status"]
 	m.Unstructured.SetResourceVersion(unstructuredFromServer.GetResourceVersion())
+	m.Unstructured.SetOwnerReferences(unstructuredFromServer.GetOwnerReferences())
```

Comment:
With this the Method Name probably isnt the best anymore, maybe something like UpdateModuleFromCluster?

Reply:
renamed